large_image/tilesource/base.py

```diff
@@ -126,14 +126,18 @@ def __init__(self, encoding='JPEG', jpegQuality=95, jpegSubsampling=0,
         self.jpegSubsampling = int(jpegSubsampling)
         self.tiffCompression = tiffCompression
         self.edge = edge
-        self._jsonstyle = style
```

Comment:
This needs to be set even if `not style`.

Reply:
🤦🏻‍♂️  added this back